### PR TITLE
[RFC][NI] Update ember to 2.16 and migrate to the new module API

### PR DESCRIPTION
**Pull request by AndreJoaquim**
NI

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,10 @@
 /coverage/*
 /libpeerconnection.log
 npm-debug.log*
+yarn-error.log
```
Comment (asantos00) [inline .gitignore:17]: I think you can do `./*.log`, I don't see us wanting to commit log files

File: addon/components/uni-modal.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import Ember from 'ember';
+import Component from '@ember/component';
+import { observer } from '@ember/object';
+import $ from 'jquery';
```
Comment (asantos00) [inline addon/components/uni-modal.js:3]: oh 😞

**Review by asantos00 — CHANGES_REQUESTED** [review]
Good job, and good job using `ember-try` 👏

**Review by asantos00 — APPROVED** [review]

**Review by mmelo — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -14,6 +14,10 @@
...
 testem.log
-.DS_Store
```
Comment (telmaantunes) [inline .gitignore]: why remove these two?

Comment (AndreJoaquim) [reply .gitignore]: Mea culpa

**Review by DaniFrancisco — APPROVED** [review]
:1st_place_medal:

**Pull request merged**